recipes/daemoniker/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "daemoniker" %}
+{% set version = "0.2.3" %}
+{% set hash = "d9b62c8d75227b3da88c0ca22b82e19d88d32dade4ffa838427235e0cdd5e578" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - daemoniker
```

can we try to add ``pip check`` here please?
```suggestion
    - daemoniker
  requires:
    - pip
  command:
    - pip check
```

---

Fixed thanks :)